test/e2e/test_operator_logs.go

```diff
@@ -208,11 +216,6 @@ func TestOperatorLogs(t *testing.T) {
 
 func isReconcilerErrorAllowedByRegexMatch(errorMsg string) bool {
 	allowedReconcilerErrorRegexes := []string{
-		// For some reason this sometimes happen on CI. While this might be an actual
-		// issue, this should not fail the test on its own.
-		//
-		// Possibly related upstream issue:
-		// - https://github.com/kubernetes-sigs/controller-runtime/issues/1881
```

Comment:
Why remove this comment? I believe leaving this in makes sense to keep readers aware of the issue.

Reply:
Seems there's a k/k issue linked in that controller-runtime issue: https://github.com/kubernetes/kubernetes/issues/124347. Can we link that instead then?

Reply:
sure, linked